dailizabot/text_patterns.py

```diff
@@ -15,6 +15,11 @@
     
     [r"Rechne .*",
     ["Ich bin ein schlechter Kopfrechner.",
-    "Damit kann ich nicht rechnen ;)"]]
+    "Damit kann ich nicht rechnen ;)"]],
+     
+    [r"Warum (.*?) (.*?) (.*)[?.!]",
+    ["Ich weiß nicht warum {1} {2} {0}.",
+    "Ich bin mir nicht sicher, warum {1} {2} {0}.",
+    "Ich finde auch, dass {1} {2} {0}."]],
```

Comment:
Hallo, die Antworten in den Zeilen 21 und 22 sind passend, weil Dailiza auf die Frage des Nutzers reagiert - auch, wenn die Frage nicht beantwortet wird, wie es der Nutzer wahrscheinlich gerne hätte :)
Die in Zeile 23 implementierte Reaktion ist keine Antwort, sondern eine Bestätigung der Feststellung.
Man könnte die Reaktion so anpassen, dass es so aussieht, als hätte Dailiza "verstanden", dass es sich um eine "Frage" handelt, z.B. mit "Das habe ich mich auch immer gefragt." :)